src/modules/CC1101/CC1101.cpp

```diff
@@ -489,8 +490,11 @@ int16_t CC1101::setRxBandwidth(float rxBw) {
       float point = (RADIOLIB_CC1101_CRYSTAL_FREQ * 1000000.0)/(8 * (m + 4) * ((uint32_t)1 << e));
       if(fabs((rxBw * 1000.0) - point) <= 1000) {
         // set Rx channel filter bandwidth
-        return(SPIsetRegValue(RADIOLIB_CC1101_REG_MDMCFG4, (e << 6) | (m << 4), 7, 4));
+        uint16_t state = SPIsetRegValue(RADIOLIB_CC1101_REG_MDMCFG4, (e << 6) | (m << 4), 7, 4);
```

Comment:
State variables should be int16_t - also, this doesn't seem different from the orignal line ...?

Reply:
ACK. Restored.